docker-compose/withPostgresAndWorker/docker-compose.yml

```diff
@@ -67,6 +66,8 @@ services:
 
   n8n-worker:
     <<: *shared
-    command: worker
+    command: n8n worker
+    environnement:
+      - EXECUTIONS_MODE=queue
```

Comment:
This bit is shared, and should be present on all containers when running in queue mode